Source/ACE.Server/WorldObjects/Managers/EmoteManager.cs

```diff
@@ -1343,6 +1343,13 @@ public void Enqueue(BiotaPropertiesEmote emoteSet, WorldObject targetObject, int
 
                         if (Nested == 0)
                             IsBusy = false;
+
+                        if (emoteSet.Category == (uint)EmoteCategory.Death)
```

Comment:
this should only run if Nested == 0

Reply:
redacted, this is using Category==Death instead of OnDeathEmoteInProgress

this should only be true in the base branch of OnDeath, so this should be fine